.github/workflows/unit-test.yml

```diff
@@ -5,7 +5,9 @@ name: Unit Tests
 on:
   pull_request:
   push:
-    branches: [master, wp-trunk]
```

Comment:
`wp-trunk` doesn't seem to exist (there is a `wp/trunk`), so the change should fix that 😄